src/components/UserProfile/Badges.jsx

```diff
@@ -73,7 +74,7 @@ export const Badges = props => {
               </span>
         
             <div >
-              {(props.canEdit || props.role == 'Owner' || props.role == 'Administrator') && (
+              {(props.canEdit || props.role == 'Owner' || props.role == 'Administrator' || canModifyBadgeAmount) && (
```

Comment:
```suggestion
              {(props.canEdit || canModifyBadgeAmount) && (
```
I'd would make it so admins/owners have the relevant permissions, but that's just me.